src/drivers/cloudwatch.rs

```diff
@@ -89,24 +89,24 @@ impl CloudWatchDriver {
             },
         }
         self.streams.insert(stream_name.to_owned());
-        return Ok(());
+        Ok(())
```

Comment:
I know Clippy's default is to include this lint (which I have expressly allowed in the Cargo.toml). It may be less rust like but in general I prefer the explicit returns. 

I will accept these changes because they do not have any other real return conditions, but since this function has an early return that may happen, I'd prefer to keep the explicit return at the end.

```suggestion
        return Ok(());
```

Reply:
@miceg by tomorrow I'll assume you don't have strong opinions on this, or based on responses from your main PR, and will make this edit and merge.

Reply:
I gave up arguing with Clippy's ideas long ago. 😅 

I've had another look at this, and it's possible to make these chains simpler.

I've also noticed that if it can't create a log group, it won't actually return an error.

I'll incorporate "explicit return if there's a return elsewhere in the function" into those changes.

Reply:
Thanks!